flink-table/flink-table-code-splitter/src/main/resources/META-INF/NOTICE

```diff
@@ -1,4 +1,4 @@
-flink-table-code-splitter
+Flink : Table : Code Splitter
```

Comment:
Why not `flink-table-code-splitter`?

Reply:
Most of the notice files use module name instead of their artifact id

Reply:
I've checked other NOTICE files. All other NOTICE files use artifact id instead of module name. So currently I'll follow these behavior.

However for the "integrated" NOTICE file generated in modules like `flink-table-uber`, modules without hand-written NOTICE files are referenced with their module name. I think there might be some conflicts here. @zentol

Reply:
After discussing with @zentol , using artifact id is the correct way. I've created FLINK-23294 for changing the title of the default NOTICE files.